libdm/libdm-deptree.c

```diff
@@ -2489,7 +2489,7 @@ static int _cache_emit_segment_line(struct dm_task *dmt,
 	EMIT_PARAMS(pos, " %s", name);
 
 	/* Do not pass migration_threshold 2048 which is default */
-	EMIT_PARAMS(pos, " %u", (seg->policy_argc + (seg->migration_threshold != 2048) ? 1 : 0) * 2);
```

Comment:
should it be ((seg->migration_threshold != 2048) ? 1 : 0))

Reply:
The current implementation is to appease `-Wparentheses` without changing program semantics, as you pointed out in https://github.com/lvmteam/lvm2/pull/56#issuecomment-893813114.  :)